apps/dashboard/src/components/primitives/editor.tsx

```diff
@@ -126,6 +126,7 @@ const baseTheme = (options: { multiline?: boolean }) =>
     // important to show the cursor at the beginning of the line
     '.cm-line': {
       marginLeft: '1px',
+      lineHeight: '20px',
```

Comment:
increased the line height to 20px, which together with padding 8px gives 36px input height

Reply:
the pill is 18px so smaller than 20px line height meaning it wont affect the height of the input